.github/workflows/beam_PostCommit_Java_Examples_Dataflow_V2.yml

```diff
@@ -49,6 +49,7 @@ env:
   DEVELOCITY_ACCESS_KEY: ${{ secrets.DEVELOCITY_ACCESS_KEY }}
   GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GE_CACHE_USERNAME }}
   GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GE_CACHE_PASSWORD }}
+  DOCKER_TAG: latest
```

Comment:
Is this "latest" tag needed? Below it sets DOCKER_TAG to a unique tag, overrides this default. And checked us.gcr.io/apache-beam-testing/github-actions/beam_java17_sdk_distroless no image has tagged "latest"

Reply:
Thanks for catching that, I’ve removed the line.